styles/main.css

```diff
@@ -110,7 +111,8 @@ div.favourites div{
     display: none;
 }
 
-div.favourites label:active{
+div.favourites label:active,
+div.favourites input:checked + label {
```

Comment:
I added this selector here so it's a bit more clear which tab is currently selected, even after you've clicked.  (:active only selects an element WHILE it's being clicked)